chart/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+appVersion: 0.1.0-SNAPSHOT
```

Comment:
There's not a good appVersion to put here. This version will need to be updated on every release. Also, this chart will need to be published somewhere for consumption like GitHub pages or an artifact registry. These can be followups.

Reply:
appVersion has been updated.